tests/unit/test_estimator.py

```diff
@@ -632,6 +632,31 @@ def test_framework_with_profiler_config_without_s3_output_path(time, sagemaker_s
     ]
 
 
+def test_framework_with_no_default_profiler_in_unsupported_region():
+    boto_mock = Mock(name="boto_session", region_name="cn-north-1")
```

Comment:
I don't think we should hardcode the region in the test. Let's just test that every region in the unsupported list passes this test.

```
from sagemaker.fwutils import PROFILER_UNSUPPORTED_REGIONS
...
@pytest.mark.parametrize("region", PROFILER_UNSUPPORTED_REGIONS)
def test_framework_with_no_default_profiler_in_unsupported_region(region):
    boto_mock = Mock(name="boto_session", region_name=region)
...
```

Reply:
That way this test isn't dependent on what exact regions aren't supported.

Reply:
+1 and also when we remove `cn-north-1` from the list, we won't have to update the test this way

Reply:
Done.